tests/unit_tests/db_engine_specs/test_doris.py

```diff
@@ -154,21 +206,29 @@ def test_get_schema_from_engine_params(
     "database_value,expected_catalog",
     [
         ("catalog1.schema1", "catalog1"),
-        ("catalog1", "catalog1"),
```

Comment:
This test is wrong. If the URI has `catalog1` for the database, eg:

```
doris://user:pass@host/catalog1
```

Then `catalog1` is the schema ("database"), not the catalog. See https://pypi.org/project/pydoris/.